src/styles/blocks/caledar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-widht;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+
+  @each $number,
+    $day in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: $number * $calendar-day-space;
+    }
+  }
+
+  @for $day from 28 through 31 {
+    &--month-length--#{$day} {
+      .calendar__day:nth-child(n + #{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: $squere-side;
+    height: $squere-side;
+    border: $calendar-day-border solid black;
+    background-color: $grey;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    transition-duration: 0.5s;
```

something is wrong with transition here: 
1. you just set transition-duration, but without any property
2. you can use transition like here: https://www.w3schools.com/css/css3_transitions.asp
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```
or 
```suggestion
    transition-property:  background-color, transform;
    transition-duration: 0.5s;
```